src/styles/app.css

```diff
@@ -0,0 +1,10 @@
+.app {
+  display: flex;
+  flex-direction: column;
+}
+
+.nasa-logo {
+  margin: auto;
+  width: 500px;
+  height: 300px;
+}
```

Comment:
I would recommend changing this to use `aspect-ratio`, a percentage `width` and a fixed `max-width` to make this logo more scallable for different browser widths, i.e.
```
.nasa-logo {
  margin: auto;
  aspect-ratio: 5 / 3;
  width: 100%;
  max-width: 500px;
}
```